### DISCUSSION TITLE
Specify keptn/version as the minimum version, version range, or something else?

### DISCUSSION THREAD

**Discussion by christian-kreuzberger-dtx** [keptn-contrib/artifacthub#102] [General]
I would like to discuss how we should handle `keptn/version` in artifacthub-pkg.yml of every integration.

Right now we use it like this:
```
   keptn/version: "0.10.0-0.13.1" 
```

This means that for every new Keptn version, we would need to update all artifacthub-pkg.yml of all integrations.

However, I believe that this version range is very important, as it indicates which version this integration was tested with. Please propsoe other ideas.

**Comment by christian-kreuzberger-dtx** [comment]
# Proposal

Have multiple fields, whereas
```
keptn/version
```
specifies a specific minimum and/or target version (no version range) that the integration is built for, and
```
keptn/testedwith
```
could specify a version range that this integration should be compatible with.

**Comment by agardnerIT** [comment]
As part of GSoC could we propose the creation of some kind of test harness where you specify the integrations you need and the events they each listen to. Whenever there's a new release of Keptn core, the workflow builds the new version, installs the service and / or sets up the web hook then tests.

Could we use Keptn on Keptn for this? GitHub Action (new release) > Keptn > Workflow to build new Keptn > generate load > quality gate ??
